core/src/components/input/input.vars.scss

```diff
@@ -1 +1,4 @@
 @import "../../themes/ionic.globals";
+
+/// @prop - How much to scale the floating label by
+$input-floating-label-scale:           0.75 !default;
```

Comment:
~ We discussed in one of our alignments we were going to do away with manually maintaining the spacing in SASS files. 

Should this be:
```
$input-floating-label-scale: 0.75 !default;
```

Reply:
Good catch, fixed